code/_onclick/item_attack.dm

```diff
@@ -58,6 +58,9 @@
 				else
 					return 1
 		var/obj/item/organ/external/O = M.get_organ(user.zone_selected)
+		if(!O)
```

Comment:
This will also give warnings to people using blunt weapons trying to hit somebody who is resting on a surgery table.

Reply:
Yeah this is annoying, I don't see a way to both give them an error message and do a check of O exists. I think I have to settle for no error message, feels kind of odd but eh.

Reply:
It'll be fixed fully in the surgery PR so don't have to put to much effort into it